include/real_time_tools/threadsafe/threadsafe_object.hxx

```diff
@@ -176,7 +176,7 @@ ThreadsafeObject<Types...>::ThreadsafeObject()
 
 template <class... Types>
 template <int INDEX>
-ThreadsafeObject<Types...>::Type<INDEX> ThreadsafeObject<Types...>::get() const
+typename ThreadsafeObject<Types...>::template Type<INDEX> ThreadsafeObject<Types...>::get() const
```

Comment:
I have to confess ```ThreadsafeObject<Types...>::template``` is obscure to me, so I'll trust you with this one

Reply:
Same for me: I followed the error message and this seems to compile, so I am happy...

Reply:
I think the `ThreadsafeObject` should be remove as they should be part of the time_series package,